synapse/app/homeserver.py

```diff
@@ -561,11 +561,16 @@ def phone_stats_home():
 
         stats["database_engine"] = hs.get_datastore().database_engine_name
         stats["database_server_version"] = hs.get_datastore().get_server_version()
-        logger.info("Reporting stats to matrix.org: %s" % (stats,))
+        logger.info("Reporting stats: %s" % (stats,))
```

Comment:
could we log the endpoint here?
```suggestion
        logger.info("Reporting stats to %s: %s" % (report_stats_endpoint, stats,))
```